CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Next
 
+## v0.6.0
+
 * Fix crash when mapping PostgreSQL enum to ruby variable in https://github.com/toptal/granite-form/pull/28
```

Comment:
Should we include these:

* https://github.com/toptal/granite-form/pull/26
* https://github.com/toptal/granite-form/pull/29

Reply:
The first one, I think yes! The second one, I think no : )